### PR TITLE
Add new terminologies to the glossary

### PR DESCRIPTION
**Pull request by complexbrains**
Hi @yasminebassil @vogelbac @likeajumprope @TheChymera, I added new terminologies to the glossary from the reference listed below, and will continue filling them in as time allows, please do not hesitate to review as necessary. Thank you.

### CODE REVIEW

File: brainhack_book/glossary.md [hunk 1]
```diff
@@ -8,15 +8,25 @@ The societal community of individuals and groups associated with education, rese
 ### Across Field
 
 ### Accessibility
+R efers to the degree of access. Defined by an end-user basis, depending on their ability to understand or reuse content.
```
Comment (TheChymera) [inline brainhack_book/glossary.md:11]: `R `, Typo.

**Comment by TheChymera** [comment]
Was there a selection process for these terms? Really most of them are “see dictionary” or “see websearch”. Ideally we would only define the terms which have a slightly derived meaning in the brainhack context. The MIT license is the MIT license, CC is Creative Commons, methodology is methodology, etc.

**Comment by complexbrains** [comment]
@TheChymera Thank you very much for your comment. Actually you are right. The beginning of the list was based on the 1st ever draft of the paper (15 pages long version) that I went through the whole text and took the terminologies from there and started to fill them in. 

But then some more added thinking this glossary might be the ultimate open science/source development community glossary. And those licencing types we use quite much for licencing the open projects, scripts, texts, tools and resources and always boggles the newcomers regarding what they mean or include. 

Therefore I wanted to add them yesterday. But all in all we do not have general criteria atm but making it a useful tool for new comers to have most basic terminologies as possible even they are not in the paper (given it is 4 page now) anymore. Does that make sense? I wonder what you would and other think about it. Again thank you 🤗

**Comment by TheChymera** [comment]
@complexbrains I would support cutting all items for the list which are defined in the same way either on an online dictionary or wikipedia (MIT and Creative Commons licenses won't pop up in the dictionary, but in fact they as well as widely accepted summaries are even easier to find with a web search).

**Review by likeajumprope — APPROVED** [review]
Fine except for the typo

**Review by vogelbac — COMMENTED** [review]
Sorry for my late response. I think it would much more readable if we do a point licenses and make sub points of licenses. The apache license for example is for me not detailed written, to make a separate point. I know that we wanted to create a glossary that includes all facts we need. but I would not list every single license. I think the best way would be to summarize the licenses and describe the differences of the type there and maybe make a short list of the most important.

**Comment by Remi-Gau** [comment]
I would suggest dropping the README and building locally guidelines otherwise this will merge conflicts with PR #51

**Comment by Remi-Gau** [comment]
> Sorry for my late response. I think it would much more readable if we do a point licenses and make sub points of licenses. The apache license for example is for me not detailed written, to make a separate point. I know that we wanted to create a glossary that includes all facts we need. but I would not list every single license. I think the best way would be to summarize the licenses and describe the differences of the type there and maybe make a short list of the most important.

Talking about linceces: this repo has a licence for "code" but not for "content", correct?

Might be better to have the content of the book be under CC-BY (or something like that) as the BSD licences we have might not be best for that.

I think we talked about that but it was never implemented. If I get a :+1: I will open an issue to keep track of that.

**Comment by Remi-Gau** [comment]
superseded by #121

**Pull request closed without merging**